Lib/enum.py

```diff
@@ -153,15 +199,29 @@ def __set_name__(self, enum_class, member_name):
         enum_member._name_ = member_name
         enum_member.__objclass__ = enum_class
         enum_member.__init__(*args)
+        enum_member._sort_order_ = len(enum_class._member_names_)
```

Comment:
this doesn't look right-- `_sort_order_` is an int?

Reply:
Yes, because sorting `int`s is easy.

Reply:
This code through me off because it's not clear how `len(enum_class._member_names_)` provides the sort order, so it may warrant a comment.